src/docs/release/breaking-changes/add-currentAutofillScope-to-TextInputClient.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Add currentAutofillScope to TextInputClient
```

Comment:
This sounds like a PR title, not a migration guide title. Maybe "New currentAutofillScope property on TextInputClient"?